pkg/synapse/synapse.go

```diff
@@ -317,26 +330,27 @@ func (s *synapse) logout() {
 // sendResourceUpdates sends resource status of synapse
 func (s *synapse) sendResourceUpdates(
 	status core.StatType,
-	runnerOpts core.RunnerOptions,
+	runnerOpts *core.RunnerOptions,
 ) {
 	specs := GetResources(runnerOpts.Tier)
 	resourceStats := core.ResourceStats{
 		Status: status,
 		CPU:    specs.CPU,
 		RAM:    specs.RAM,
 	}
+	s.logger.Debugf("sending resource update to lambdatest %+v", resourceStats)
```

Comment:
can we add build ID here to track these logs ?

Reply:
added `jobID` and `buildID`